buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -464,15 +464,15 @@ class BeamModulePlugin implements Plugin<Project> {
     def classgraph_version = "4.8.104"
     def errorprone_version = "2.10.0"
     // Try to keep gax_version consistent with gax-grpc version in google_cloud_platform_libraries_bom
-    def gax_version = "2.16.0"
+    def gax_version = "2.18.7"
     def google_clients_version = "1.32.1"
```

Comment:
Can you upgrade this `google_clients_version` to 2.0.0? As per  https://github.com/googleapis/java-shared-dependencies/blob/v3.0.1/first-party-dependencies/pom.xml#L69. This is for `com.google.api-client:google-api-client` in line 574. ([Maven link](https://search.maven.org/artifact/com.google.api-client/google-api-client/2.0.0/jar)).

Reply:
Then, update line 578 - 584 as followings:

```
        google_api_services_bigquery                : "com.google.apis:google-api-services-bigquery:v2-rev20220827-$google_clients_version",
        google_api_services_clouddebugger           : "com.google.apis:google-api-services-clouddebugger:v2-rev20220318-$google_clients_version",
        google_api_services_cloudresourcemanager    : "com.google.apis:google-api-services-cloudresourcemanager:v3-rev20220828-$google_clients_version",
        google_api_services_dataflow                : "com.google.apis:google-api-services-dataflow:v1b3-rev20220812-$google_clients_version",
        google_api_services_healthcare              : "com.google.apis:google-api-services-healthcare:v1-rev20220818-$google_clients_version",
        google_api_services_pubsub                  : "com.google.apis:google-api-services-pubsub:v1-rev20220829-$google_clients_version",
        google_api_services_storage                 : "com.google.apis:google-api-services-storage:v1-rev20220705-$google_clients_version",
```

The "google-api-services-*" libraries need to have "-2.0.0" suffix version to work with google-api-client version 2.0.0. (Details https://github.com/googleapis/google-api-java-client-services/issues/13338 )

Reply:
Incorporated the changes.

Reply:
There are a lot of failed github checks after this. Can you please check this?

Reply:
What is error messages?

Reply:
```
/home/runner/work/beam/beam/sdks/java/extensions/google-cloud-platform-core/src/main/java/org/apache/beam/sdk/extensions/gcp/options/GcpOptions.java:27: error: package com.google.api.services.cloudresourcemanager does not exist
import com.google.api.services.cloudresourcemanager.CloudResourceManager;
                                                   ^
/home/runner/work/beam/beam/sdks/java/extensions/google-cloud-platform-core/src/main/java/org/apache/beam/sdk/extensions/gcp/options/GcpOptions.java:28: error: package com.google.api.services.cloudresourcemanager.model does not exist
import com.google.api.services.cloudresourcemanager.model.Project;
```